### PR TITLE
Added Dockerfile

### PR DESCRIPTION
**Pull request by eiabea**
- Dockerfile for building from master branch
- Updated readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +72,22 @@ Packaging for your favorite distribution would be a welcome contribution!
 
 ## Compiling Monero from Source
 
+### Docker
```
Comment (radfish) [inline README.md]: Thanks for the docker script. Could you please move this section as an item in the list of "Installing Monero from Package"? Docker fits into the same category as a PKGBUILD script (AUR) or a homebrew script (OS X).

**Comment by eiabea** [comment]
@radfish you are right! i moved it up

**Comment by radfish** [comment]
Cool, thanks! I don't mean to be a pest... to make it concise I would compress the item into just the the three commands with # comments:
```
* Docker: build Monero docker container from the latest master branch of the git project and run:
        docker ...
        # either run in foreground:
         ....
        # or run in background
        ...
```

**Comment by radfish** [comment]
Oh, btw, would be good to squash these commits into one.

**Comment by eiabea** [comment]
@radfish i unified the package section, but i think i messed up commit squashing...
since now i manged to do everything without git rebase :smile:

**Comment by radfish** [comment]
Ok, squashing only takes a moment -- here's a starting point:
https://github.com/monero-project/bitmonero/pull/966#issuecomment-242885452

**Comment by fluffypony** [comment]
Closed in favour of #1028

**Pull request closed without merging**